### DISCUSSION TITLE
検索結果画面のフォーカスについて

### DISCUSSION THREAD

**Discussion by cia-rana** [increments/qiita-discussions#97] [要望・アイデア・不具合]
検索結果画面（ https://qiita.com/search ）では初期フォーカスが検索ワード入力欄になっていますが、これを止めていただきたいです。

というのも、検索直後や、検索結果画面から記事ページに行って帰ってきたとき、2ページ目以降を閲覧したときに再度検索ワードを変更することは少ないからです。
さらに検索結果画面の下側にスクロールするためにキーボードの↓ボタンを押す際に、検索ワード入力欄にフォーカスが当たっているためにスクロールできない問題もあるからです。

**Comment by kskwtnk** [comment]
@cia-rana 
ご意見ありがとうございます！

こちらはアクセシビリティなどの観点から完全に良くありませんね
我々も見落としていました……ありがとうございます！

スケジュール的に年明けになってしまいそうですが、対応いたします！

**Reply by kskwtnk** [reply]
https://github.com/increments/qiita-discussions/issues/98

取り急ぎ、Issueを作成しました！

**Comment by kskwtnk** [comment]
@cia-rana 
こちら、修正してリリース完了いたしました！
